share/rpcuser/rpcuser.py

```diff
@@ -36,6 +36,6 @@
 m = hmac.new(bytearray(salt, 'utf-8'), bytearray(password, 'utf-8'), digestmod)
 result = m.hexdigest()
 
-print("String to be appended to bitcoin.conf:")
+print("String to be appended to dogecoind.conf")
```

Comment:
On my machine, this file is `dogecoin.conf`. Can you confirm on yours?

Reply:
I see it as 'dogecoind.conf' in dogecoin/contrib/init/

Reply:
It's a different config file. The correct file referenced to here is https://github.com/dogecoin/dogecoin/blob/master/contrib/debian/examples/dogecoin.conf